### PR TITLE
Accelerator

### PR DESCRIPTION
**Pull request by paulbalaji**
Add function to return arguments for Unity Accelerator, if `ACCELERATOR_ENDPOINT` environment variable is provided.

### CODE REVIEW

File: scripts/pinned-tools.sh [hunk 1]
```diff
@@ -83,3 +83,7 @@ function isDocsBranch() {
   fi
   return 1
 }
+
+function getAcceleratorArgs() {
+  echo ${ACCELERATOR_ENDPOINT:+-adb2 -enableCacheServer -cacheServerEnableDownload true -cacheServerEnableUpload true -cacheServerEndpoint "${ACCELERATOR_ENDPOINT}"}
```
Comment (paulbalaji) [inline scripts/pinned-tools.sh:88]: I spent a day and a half figuring out why nothing was working until I stumbled upon a forum post where someone says to add `-adb2` if trying to access Accelerator in batch mode

:tableflip:

Comment (zeroZshadow) [reply scripts/pinned-tools.sh:88]: Could it be some of our projects have not been converted to asset db 2?

Comment (paulbalaji) [reply scripts/pinned-tools.sh:88]: I observed this on all projects

Comment (jamiebrynes7) [reply scripts/pinned-tools.sh:88]: We did do the asset db upgrade & then a full asset refresh during the 2019.3 upgrade though.

Comment (paulbalaji) [reply scripts/pinned-tools.sh:88]: Fresh clones also show asset db 2 is used in the preferences

**Review by jamiebrynes7 — APPROVED** [review]

**Review by zeroZshadow — APPROVED** [review]

**Pull request merged**